README.md

```diff
@@ -71,16 +71,24 @@ The easiest way to get started with Thunder, requiring no extra installations or
 
 &#160;
 
+
 ## Install Thunder
 
-To use Thunder on your local machine, first install [nvFuser](https://github.com/NVIDIA/Fuser) nightly and PyTorch nightly together as follows:
+To use Thunder on your local machine:
+- install [nvFuser](https://github.com/NVIDIA/Fuser) nightly and PyTorch nightly together as follows:
 
 ```bash
 # install nvFuser which installs the matching nightly PyTorch
 pip install --pre 'nvfuser-cu121[torch]' --extra-index-url https://pypi.nvidia.com
 ```
+- install [cudnn](https://gitlab-master.nvidia.com/cudnn/cudnn_frontend) as follows:
```

Comment:
added cudnn install instructions to "Install Thunder" section of root level README.MD

Reply:
do we need to add cudnn-fe to requirements.txt?

Reply:
I did not see nvfuser in requirements.txt. And only see nvfuser mentioned here in readme.md.
So I followed the same with cudnn.
Let me know if its fine to add cudnn in requirements.txt.

CC @Borda

Reply:
Yeah @Borda's input would be great here!

Reply:
I don't know why nvfuser is not part of the requirements.txt but we should add cuDNN there.

Reply:
I think we want things to be installable and runnable on CPU machines without the libs, but maybe we could add requirements/gpu.txt or so...

Reply:
Are CPU PyTorch wheels hosted on PyPI? PyTorch's webpage (https://pytorch.org/get-started/locally/) suggests using `--index-url https://download.pytorch.org/whl/cpu`.

Reply:
Added cudnn to requirements/base.txt now.
made a new feature request for nvfuser discussion. #438 

Lets mark this conversation resolved?

Reply:
Actually reverted the change to add cudnn to requirements.txt.
There are no pip wheels for windows and that is causing CI failures.

Reply:
Gotcha.
@t-vi makes the call but I don't think windows support for cuDNN is critical right now.

marking as resolved, Tom feel free to reopen if you want to see a different solution.